### DISCUSSION TITLE
app.get_edit_app_store_version || app.get_latest_app_store_version || app.get_live_app_store_version ，What exactly did you use App Store Connect API

### DISCUSSION THREAD

**Discussion by DreamPWJ** [fastlane/fastlane#19987] [Q&A]
Other than ruby, are you going to provide a node version, because node is used by more people
![image](https://user-images.githubusercontent.com/16730031/154593958-7d6bfd72-db5f-4bd7-8f27-2aacc02aeaa5.png)
appInfos The interface could not get the version being submitted for review。

[a comment by an automated account is not shown]

**Comment by DreamPWJ** [comment]
I have been asking questions in the question area for several months and no one has returned. Thank you for your answer

**Comment by lucgrabowski** [comment]
I don't think that node version is planned, but you're more than welcome to add pull requests with it. 
Please remember that this is an open source project and people work on it in their free time.

**Reply by DreamPWJ** [reply]
get_edit_app_store_version  What exactly did you use App Store Connect API？

**Reply by DreamPWJ** [reply]
I now want to get the status of my app from approval to launch for notification
